lighthouse-core/gather/gather-runner.js

```diff
@@ -90,11 +91,19 @@ class GatherRunner {
       .then(_ => {
         return driver.cleanAndDisableBrowserCaches();
       }).then(_ => {
+        // Check if multiple tabs of origin are open
```

Comment:
yeah, I now think this check definitely be moved up first. No point in getting started if it's not going to work out. Maybe just:
```js
return driver.checkForMultipleTabsAttached(options.url)
  .then(_ => {
    // Enable emulation if required.
    return options.flags.mobile && driver.beginEmulation();
  }).then(_ => {
    return driver.cleanAndDisableBrowserCaches();
  }).then(_ => {
    // Clears storage for origin.
    return driver.clearDataForOrigin(options.url);
  });
```
?